lib/src/query.dart

```diff
@@ -5,10 +5,10 @@ import 'package:deno_postgres_interop/src/result_type.dart';
 
 /// [deno-postgres@v0.17.0/Query](https://deno.land/x/postgres@v0.17.0/query/query.ts?s=Query).
 @JS()
-class Query<T extends ResultType> {}
+class Query {}
```

Comment:
I'm not sure, but there might some other places that have similar generics that can't be translated to dart correctly, so there might be some places to be improved

It is just one of the ways to read this value, so it is okay